rapids/rapids.sh

```diff
@@ -124,85 +154,51 @@ EOF
   fi
 }
 
-function create_conda_env() {
-  echo "Create RAPIDS Conda environment..."
-  # For use with Anaconda component
-  local -r conda_env_file="${BUILD_DIR}/conda-rapids-env.yaml"
-  cat <<EOF >"${conda_env_file}"
-name: ${RAPIDS_ENV}
-channels:
-  - rapidsai/label/xgboost
-  - rapidsai
-  - nvidia
-  - conda-forge
-dependencies:
-  - cudatoolkit=${CUDA_VERSION}
-  - rapids=${RAPIDS_VERSION}
-  - gcsfs
-  - dill
-  - ipykernel
-EOF
-  conda config --set channel_priority flexible
-  conda env create --file "${conda_env_file}"
-}
-
-function install_conda_kernel() {
-  conda install -y nb_conda_kernels
-  # Restart Jupyter service to pickup RAPIDS environment.
-  service jupyter restart || true
-}
+configure_systemd_dask_service() {
+  echo "Configuring systemd Dask service for RAPIDS..."
+  local -r dask_worker_local_dir="/tmp/dask"
+  local -r conda_env_bin=$(conda info --base)/bin
```

Comment:
This we need to split on declaration and assignment, because it executes command during initialization.

Reply:
Ack